lib/logging.dart

```diff
@@ -52,7 +52,7 @@ class Logger {
   /// Singleton constructor. Calling `new Logger(name)` will return the same
   /// actual instance whenever it is called with the same string name.
   factory Logger(String name) {
-    return _loggers.putIfAbsent(name, () => new Logger._named(name));
+    return _loggers.putIfAbsent(name, () => Logger._named(name));
```

Comment:
While you're here, perhaps the short form/ . (`=>`?)

Reply:
Trying to stick w/ just lints here – to make review easier.

Happy to do that in a follow-up